tests/unit/lexicon/test_medium.py

```
@@ -185,6 +185,49 @@ def test_the_refusal_belongs_to_the_medium_rite():
 
 
+def _summary_of(name: str, grimoire: Grimoire) -> str | None:
+    began = next(
+        e for e in grimoire.events if isinstance(e, RiteBegan) and e.name == name
+    )
+    return began.summary
+
+
+class TestMediumSummary:
+    @staticmethod
+    def test_the_first_string_argument_on_one_line():
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `test_the_first_string_argument_on_one_line`

Add return type annotation: `None`

(ANN205)